core/chaincode/handler.go

```diff
@@ -678,6 +686,46 @@ func (h *Handler) HandleGetState(msg *pb.ChaincodeMessage, txContext *Transactio
 	return &pb.ChaincodeMessage{Type: pb.ChaincodeMessage_RESPONSE, Payload: res, Txid: msg.Txid, ChannelId: msg.ChannelId}, nil
 }
 
+// Handles query to ledger to get state
```

Comment:
Function comments should start with the function name, e.g. 
`// HandleGetStateMultipleKeys queries multiple keys in one batch request`

Reply:
fixed fixed